src/placement/tests/jump_map_place_obj.c

```diff
@@ -1607,7 +1607,7 @@ placement_handles_multiple_states(void **state)
 	 */
 	ctx.ver = ver_after_fail;
 	jtc_scan(&ctx);
-	assert_int_equal(ctx.rebuild.out_nr, 1);
+	assert_int_equal(ctx.rebuild.out_nr, 2);
```

Comment:
why 2 shards changed? and the comment in L1606 need update.

Reply:
Oh, because we compare the shard to know how many shard needs to be rebuilt, so the shard might need to be rebuilt even it is healthy, due to satisfy the co-location or RF etc, i.e. there might be more shard needs to be rebuilt in some rare cases.